### ISSUE TITLE
Incorrect namespaces in OAuth2Handler

### ISSUE THREAD

**Issue by kurakin-oleksandr** [kamermans/guzzle-oauth2-subscriber#12]
in `OAuth2Handler`:

```
public function __construct(
                    GrantType\GrantTypeInterface $grantType,
                    GrantType\GrantTypeInterface $refreshTokenGrantType = null,
                    Signer\ClientCredentials\ClientCredentialsSigner $clientCredentialsSigner = null,
                    Signer\AccessToken\AccessTokenSigner $accessTokenSigner = null
    ) {
```

namespaces `Signer\ClientCredentials\ClientCredentialsSigner` and `Signer\AccessToken\AccessTokenSigner` are incorrect.

As workaround can be used setters.

**Comment by kamermans** [comment]
Thanks for the report!  This is already fixed in the latest version.  Are you able to confirm that it is working as expected?

**Comment by kamermans** [comment]
Ah, that's the issue then - I'll make another stable release after I review the changes and the test suite coverage today!

**Comment by kurakin-oleksandr** [comment]
thanks!

**Comment by kamermans** [comment]
I have released `v1.0.5` stable: https://packagist.org/packages/kamermans/guzzle-oauth2-subscriber#v1.0.5

Let me know if you have any other issues, otherwise I'll assume this issue is resolved!
